src/lib/containers/synapse_table_functions/SynapseTableCell.tsx

```diff
@@ -38,6 +38,9 @@ type SynapseTableCellProps = {
   columnModels?: ColumnModel[]
   tableEntityId?: string
   rowData: string[]
+  isEntityView: boolean
+  rowId: number
+  rowVersionNumber: number
```

Comment:
These should be optional, right?

Reply:
Actually, we'll always have those, but they may not be useful, so they need not be optional. Nvm!